### DISCUSSION TITLE
Anyone else noticing component update issues with 10.19.3? (Solved -- my fault)

### DISCUSSION THREAD

**Discussion by jaredkrinke** [preactjs/preact#4249] [General]
Edit: I was having an issue with HTML tables that worked fine in Preact 10.17.1 and 10.18.2, but was not rendering correctly in 10.19.3. After trying to reduce the repro to something minimal, I found that (in one particular case), my `render()` was returning a `<table>` with 2 `<thead>`'s. Oops! After fixing that issue, everything's working fine again, even on 10.19.3.

Initially, I thought it was a regression in Preact, but it was just my own fault. Nothing to see here! :)

(I'm leaving this up in case anyone else runs into a similar situation when upgrading, and also in case someone already read my original message and wants to see the resolution.)

**Comment by jaredkrinke** [comment]
One last note: I (originally) setup Preact Devtools and it didn't flag any issues with my table, even though it contained two `<thead>` elements. I kind of thought this would get flagged based on this note in the docs:

> Improper nesting of table detected
> 
> HTML has a very clear directions on how tables should be structured. Deviating from that will lead to rendering errors that are very hard to debug. In Preact we'll detect this and print an error. To learn more about how tables should be structured we can highly recommend [the mdn documentation](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics)

If anyone knows whether or not this should have been flagged as a warning with Preact Devtools, let me know and I can follow up.

**Reply by rschristian** [reply]
The warnings are limited to improper nesting specifically, such as using a `<thead>` or `<tr>` outside of a `<table>`, and therefore don't cover your situation. That's "working as intended", not a bug. 

However, I think that'd make a pretty reasonable feature request,  so feel free to open an issue for that!
